cicecore/cicedynB/general/ice_flux_bgc.F90

```diff
@@ -138,22 +138,22 @@ subroutine alloc_flux_bgc
          fdust       (nx_block,ny_block,max_blocks), & ! ice-ocean dust flux (kg/m^2/s), positive to ocean
          hin_old     (nx_block,ny_block,ncat,max_blocks), & ! old ice thickness
          dsnown      (nx_block,ny_block,ncat,max_blocks), & ! change in snow thickness in category n (m)
-         HDO_ocn    (nx_block,ny_block,max_blocks), & ! seawater concentration of HDO (kg/kg)
-         H2_16O_ocn (nx_block,ny_block,max_blocks), & ! seawater concentration of H2_16O (kg/kg)
-         H2_18O_ocn (nx_block,ny_block,max_blocks), & ! seawater concentration of H2_18O (kg/kg)
-         Qa_iso     (nx_block,ny_block,icepack_max_iso,max_blocks), & ! isotope specific humidity (kg/kg)
-         Qref_iso   (nx_block,ny_block,icepack_max_iso,max_blocks), & ! 2m atm reference isotope spec humidity (kg/kg)
-         fiso_atm   (nx_block,ny_block,icepack_max_iso,max_blocks), & ! isotope deposition rate (kg/m^2 s)   
-         fiso_evap  (nx_block,ny_block,icepack_max_iso,max_blocks), & ! isotope evaporation rate (kg/m^2 s)
-         fiso_ocn   (nx_block,ny_block,icepack_max_iso,max_blocks), & ! isotope flux to ocean  (kg/m^2/s)
+         HDO_ocn     (nx_block,ny_block,max_blocks), & ! seawater concentration of HDO (kg/kg)
+         H2_16O_ocn  (nx_block,ny_block,max_blocks), & ! seawater concentration of H2_16O (kg/kg)
+         H2_18O_ocn  (nx_block,ny_block,max_blocks), & ! seawater concentration of H2_18O (kg/kg)
+         Qa_iso      (nx_block,ny_block,icepack_max_iso,max_blocks), & ! isotope specific humidity (kg/kg)
```

Comment:
Did you reintroduce trailing spaces here?

Reply:
I just shifted over the () to align better.  The blank was added to the middle of the line.